app/views/invoices/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @invoices.each do |invoice| %>
+  <%= link_to "#{invoice.invoice_id}", "/merchants/#{params[:merchant_id]}/invoices/#{invoice.id}" %>
```

Comment:
I would highly recommend to use a path helper here, this would be `merchant_invoice_path(@merchant, invoice)` In the controller scope `@merchant = Merchant.find(params[:merchant_id]`